perses/rjmc/topology_proposal.py

```diff
@@ -2366,18 +2374,23 @@ def _constraint_repairs(atom_map, old_system, new_system, old_topology, new_topo
             elif atom2 in new_hydrogens:
                 new_constraints[atom2] = length
 
-         #iterate through the atom indices in the new_to_old map, check bonds for pairs, and remove appropriate matches
+        # iterate through the atom indices in the new_to_old map, check bonds for pairs, and remove appropriate matches
         to_delete = []
```

Comment:
Suggest we instead return the old -> new atom mappings that were removed from the `atom_map`, or potentially even return an `AtomMap` object that contains only the removed mapped atoms.

Reply:
Closing. Issue created in https://github.com/choderalab/perses/issues/1129